driver-pulsar/src/main/java/io/nosqlbench/driver/pulsar/exception/PulsarDriverUnexpectedException.java

```diff
@@ -0,0 +1,9 @@
+package io.nosqlbench.driver.pulsar.exception;
+
+public class PulsarDriverUnexpectedException extends RuntimeException {
+
```

Comment:
Unexpected what?

Reply:
That's generic catch-all unexpected.

Reply:
The added functionality has some urgency. If possible, please merge these changes as soon as we can. I can change names if needed later.